README.md

```diff
@@ -37,6 +37,49 @@ For static use cases, wRPC provides [WIT] binding generators for:
 
 wRPC fully supports the unreleased native [WIT] `stream` and `future` data types along with all currently released WIT functionality.
 
+## Quickstart
+
+Here is a guide demonstrating how to bootstrap the WASM component based example implemented by
+[examples/rust/hello-component-server](examples/rust/hello-component-server) and 
+[examples/rust/hello-component-client](examples/rust/hello-component-client).
+
+### Requirement
+- docker >= 24.0.6, build ed223bc
+- rust >= 1.80.1
+
+### How-To
+1. Add the `wasm32-wasi` target
+    ```bash
+    rustup target add wasm32-wasi
+    ```
+1. Build the server WASM component
+    ```bash
+    cargo build --release -p hello-component-server --target wasm32-wasi
+    ```
+    > Output is target/wasm32-wasi/release/hello_component_server.wasm.
+1. Build the client WASM component
+    ```bash
+    cargo build --release -p hello-component-client --target wasm32-wasi
+    ```
+    > Output is target/wasm32-wasi/release/hello-component-client.wasm.
+1. Run NATS service using docker
+    ```bash
+    docker run -it --name nats-server -p 4222:4222 nats:2.10.20-alpine3.20
+    ```
+1. Serve the server WASM component
+    ```bash
+    cargo run --release --bin wrpc-wasmtime-nats -- serve -g hello-world hello world ./target/wasm32-wasi/release/hello_component_server.wasm
```

Comment:
```suggestion
    cargo run --release --bin wrpc-wasmtime-nats -- serve -g hello rust rust ./target/wasm32-wasi/release/hello_component_server.wasm
```

Would be more aligned with other examples (currently, native Go examples listen on `go` prefix and native Rust examples listen on `rust` prefix by default). `hello` is the name of the example app